multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -147,12 +147,22 @@ func (c *Cluster) Create(stackBody string) error {
 		return err
 	}
 
+	tags := make([]*cloudformation.Tag, len(c.StackTags))
+	i := 0
+	for k, v := range c.StackTags {
+		key := k
+		value := v
```

Comment:
bah forgot they were string pointers not strings, was thinking they were being copied when tag was created, you're right